.travis.yml

```diff
@@ -31,16 +31,16 @@ matrix:
 jobs:
   include:
     # Coverage tracking is slow with pypy, skip it.
-    - env: TOXENV=pypy PYTEST_NO_COVERAGE=1
+    - env: TOXENV=pypy-xdist PYTEST_NO_COVERAGE=1
```

Comment:
i believe we need to ensure lsof on pypy due to gc differences in order to catch unwanted issues

Reply:
Ok.

This is not related to pypy-xdist failing on Windows though, is it?
https://ci.appveyor.com/project/pytestbot/pytest/builds/22239624/job/2hns6pgk71weay26

Reply:
no, thats probably an issue with execnet